MeetingNMusae/MeetingNMusae/Views/MissionCardView.swift

```diff
@@ -14,84 +14,106 @@ struct MissionCardView: View {
                 .scaledToFit()
             Text(roleName)
-//                .font(.largeTitle).bold() // 이런 류는 나중에 시간 있으면
-                .font(.system(size: 24, weight: .bold))
-                .padding(20)
-
-            // 코지의 라인 익스텐션 사용
-            Line().stroke(style: StrokeStyle(lineWidth: 3, dash: [10]))
-                .frame(height: 1)
-                .padding(.vertical, 16)
-            
-            // 아래부분 회색 넣기는 나중에..? checkbox에 foreground로 하거나 clipped로 될까?
+                .font(.title2)
+                .bold()
+                // .font(.system(size: 24, weight: .bold))
+                .padding(.bottom)
+                .offset(y: -20)
             
-            HStack {
-                Text("**미션**")
-                    .font(.system(size: 18, weight: .bold)) // MARK: here (extrabold 여야함)
-
-                Spacer()
-            }
-            .padding(.vertical, 12.735)
-            .padding(.horizontal)
-            
-            // 따로 뷰로 빼서 CheckBoxView와 옆에 텍스트 같이 뒀더니 미션이 안 보여서 되돌림
-            ForEach(0...2, id: \.self) { ind in
-                HStack {
-                    // MARK: 모르겠음: userViewModel.deviceUser?가 항상 nil이 나옴
-                    Text("\(Image(systemName: progress[ind] ? "checkmark.square.fill" : "square"))")
-                        .bold()
-                        .foregroundColor(Color(UIColor.black))
-
-                    Text((self.missionViewModel.missionStrs.count > ind) ? self.missionViewModel.missionStrs[ind] : "mcnt: \(self.missionViewModel.missionStrs.count)")
-                        .font(.system(size: 16, weight: .medium))
+            ZStack {
+                Rectangle()
+                    .foregroundColor(Color.bgGray)
+                    .cornerRadius(12, corners: [.bottomLeft, .bottomRight])
+                    .padding([.horizontal, .bottom], 1.5)
+                VStack {
+                    // 코지의 라인 익스텐션 사용
+                    Line().stroke(style: StrokeStyle(lineWidth: 3, lineCap: .round, dash: [8, 10]))
+                        .frame(height: 1)
                     Spacer()
                 }
-                .padding(.vertical, 6.1)
-                .padding(.horizontal)
-                .onTapGesture {
-                    progress[ind].toggle()
-                    // 파이어베이스 update
-                    userViewModel.updateMissionProgress(roomCode: roomCode, missionId: ind, nickname: nickname)
+                VStack(alignment: .leading) {
+                    Text("미션")
+                        .font(.headline)
+                        .fontWeight(.black)
+                    //                    .font(.system(size: 18, weight: .black))
+//                        .padding(.vertical, 12.735)
+                    //                    .padding(.horizontal)
+                    
+                    // 따로 뷰로 빼서 CheckBoxView와 옆에 텍스트 같이 뒀더니 미션이 안 보여서 되돌림
+                    ForEach(0...2, id: \.self) { ind in
```

Comment:
`ind`는 변수의 역할을 알기 모호한 축약어인거 같아요. Api Design Guideline의 의거하여 `index` 와 같이 전체 이름을 사용하기를 권해드려요

```
Include all the words needed to avoid ambiguity for a person reading code where the name is used.
```